src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -153,17 +164,59 @@ export const styles = StyleSheet.create({
   textAct: {
     color: '#FF2E63',
     fontSize: 14
+  },
+  mr7: {
+    marginRight: dimen.normalizeDimen(12)
+  },
+  imageUser: {
+    height: normalize(48),
+    width: normalize(48),
+    borderRadius: normalize(24)
+  },
+  parentContact: {
+    height: normalize(72)
   }
 });
 
 const SampleChatInfoScreen = () => {
-  const {channelInfo, goBack, onContactPressed} = useAnonymousChatInfoScreenHook();
-
+  const {
+    channelInfo,
+    goBack,
+    onContactPressed,
+    selectedUser,
+    handlePressPopup,
+    handleCloseSelectUser,
+    openModal,
+    isAnonymousModalOpen,
+    blockModalRef,
+    handleShowArrow,
+    loadingChannelInfo
+  } = useChatInfoScreenHook();
   const [isLoadingMembers] = React.useState<boolean>(false);
   // TODO: Change this into useUserAuthHook
   const {signedProfileId} = useProfileHook();
-
-  const showImageProfile = () => {
+  const [profile] = React.useContext(Context)?.profile;
+  const {params}: any = useRoute();
+  const ANONYMOUS_USER = 'AnonymousUser';
+  const {anon_user_info_color_code, anon_user_info_emoji_code} =
+    channelInfo?.rawJson?.channel || {};
+  const showImageProfile = (): React.ReactNode => {
+    if (channelInfo?.channelType === CHANNEL_GROUP) {
+      return (
+        <ChannelImage>
+          <ChannelImage.Big type={GROUP_INFO} image={channelInfo?.channelPicture} />
+        </ChannelImage>
+      );
+    }
+    if (anon_user_info_color_code) {
+      return (
+        <AnonymousIcon
+          color={anon_user_info_color_code}
+          emojiCode={anon_user_info_emoji_code}
+          size={normalize(100)}
+        />
+      );
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `SampleChatInfoScreen` function has been updated with new logic for handling chat info, including a new hook `useChatInfoScreenHook`, a new state variable `isLoadingMembers`, and a new function `showImageProfile`. Ensure that these changes work as expected and handle all edge cases.



<!-- This is an auto-generated comment by OSS CodeRabbit -->